modules/API/pages/built-in-endpoints.adoc

```diff
@@ -835,39 +845,44 @@ This endpoint takes a token and its associated secret and refreshes the lifetime
 | Token to refresh.
 
 | `secret`
-| Yes
+| Yes if username and userpassword are not suplied
 | User's secret used to generate the token.
 
 | `lifetime`
-| Yes
+| No
 | Period of time for which the token is valid measured in seconds.
 |===
 
+
+=== Delete a token
+
+`DELETE /requesttoken`
+
+A user can delete a token using either
+
+* A secret, which is associated with the token to be deleted
+* Their username and password in their request header
+
 ==== Sample request
 
 [source,bash]
 ----
-curl -X PUT "http://localhost:9000/requesttoken?lifetime=15&secret=ksdoilrvpl0r0tef3d4abbpgu0t2u5la&token=0mq98l9pderkaivndf820gudg923p3l0"|jq .
+curl -X DELETE http://localhost:9000/requesttoken \
+     -d '{"secret": "your_secret_value", "token": "your_token_value"}' 
+
+curl --user example_username:example_password \
+     -X DELETE http://localhost:9000/requesttoken \
+     -d '{"token": "your_token_value"}'
+
 {
   "code": "REST-0000",
-  "expiration": 15,
+  "expiration": -1,
   "error": false,
-  "message": "Refresh token successfully.",
+  "message": "Drop token successfully.",
   "token": "0mq98l9pderkaivndf820gudg923p3l0"
 }
 ----
 
-[CAUTION]
-====
-*Known bug*: The output shows the lifetime instead of the expiration time.
-====
```

Comment:
After some trials, it seems that the output shows the expiration time although it is a little bit weird to me. 
If I refresh the same token in sequence, I can get the following output. 

Without lifetime -> "expiration":1643995777
"lifetime": 15 -> "expiration":1641403829
"lifetime": 15 -> "expiration":1641403837
"lifetime": 15 -> "expiration":1641403883
"lifetime": 15 -> "expiration":1641403902
"lifetime": 15 -> "expiration":1641404037
"lifetime": 15000 -> "expiration":1641419046
"lifetime": 150000 -> "expiration":1641554169

From the output I get, I guess it might be a precision error.